exporter/awskinesisexporter/encoding/encoder_jaeger_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encoding_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/consumer/pdata"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awskinesisexporter/encoding"
+)
+
+func TestEncodingTraceData(t *testing.T) {
+	t.Parallel()
```

Comment:
Why do we need this?

Reply:
While these currently do _effectively_ nothing, my intention with a follow up PR is abstract the kinesis producer so that we can allow for mocking of the producer and validate how much data is being written to Kinesis.

Reply:
I meant the `t.Parallel()` :) they don't do any harm but it's not a common practice in this repository AFAIK

Reply:
Oh really? I added parallel because I'm impatient and like test results quick.